### PR TITLE
Us112725/style the transcript

### PR DESCRIPTION
**Pull request by jwong-d2l**
This is a screenshot from the transcript page with print css enabled: 

image

This is a screenshot from the printed version (using the demo): 

image

Note that the `d2l-html-editor` isn't working properly in the demo page and that's why you only see a line instead of the answer. I tried to debug the issue but it was taking too long and I wanted to merge this rather quickly to unblock other work itens. However, my setup and use of the component seems to be correct given that it's working on the other page.

### CODE REVIEW

File: demo/report.html [hunk 1]
```diff
@@ -1,10 +1,20 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-gb" style="font-size:20px; font-family: 'Lato', sans-serif;">
```
Comment (TomMichielsD2L) [inline demo/report.html]: Will these be given to use in bsi land?

File: demo/report.html [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 	<head>
 		<script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 		<script type="module">
 			import '../src/components/cpd-record-report';
 		</script>
+		<script type="module" src="../../core/components/typography/typography.js"></script>
```
Comment (TomMichielsD2L) [inline demo/report.html]: Do we also get these in BSI land?

File: src/components/cpd-record-report.js [hunk 3]
```diff
@@ -36,7 +41,67 @@ class CpdRecordReport extends BaseMixin(LitElement) {
...
+			}
+			.qa-text {
+				grid-column: span 2;
+				margin: 20px;
+			}
+			d2l-html-editor {
+				border-radius: 0.3rem;
+				border-style: solid;
+				border-width: 1px;
+				border-color: initial;
```
Comment (TomMichielsD2L) [inline src/components/cpd-record-report.js]: I thought we did not want a border on the html editor?

File: src/components/cpd-record-report.js [hunk 4]
```diff
@@ -36,7 +41,67 @@ class CpdRecordReport extends BaseMixin(LitElement) {
...
+				margin-top: 60px;
+				margin-bottom: 20px;
+			}
+			p.attachment {
+				margin-top: 0.2rem;
+			}
+			@media print {
+				.container {
+					padding: 0px;
+					width: 7.5in;
```
Comment (TomMichielsD2L) [inline src/components/cpd-record-report.js]: Nice that you got print working with only a little bit of css

File: demo/report.html (hunk 1 above)
Comment (jwong-d2l) [reply demo/report.html]: Font sizes and families are provided by the blank page renderer.

File: demo/report.html (hunk 2 above)
Comment (jwong-d2l) [reply demo/report.html]: yep!

File: src/components/cpd-record-report.js (hunk 3 above)
Comment (benyamin-noori) [reply src/components/cpd-record-report.js]: You're correct. I also reduced the top/bottom margins to make it look more like the sample transcript.

File: src/components/progress-subject.js [hunk 5]
```diff
@@ -35,6 +35,9 @@ class ProgressSubject extends BaseMixin(LitElement) {
 			.progress-label {
 				vertical-align: middle;
 			}
+			d2l-meter-linear {
+				-webkit-print-color-adjust: exact;
```
Comment (jwong-d2l) [inline src/components/progress-subject.js]: this is to fix safari?

Comment (benyamin-noori) [reply src/components/progress-subject.js]: Chrome, actually!

File: src/components/cpd-record-report.js [hunk 6]
```diff
@@ -80,47 +147,99 @@ class CpdRecordReport extends BaseMixin(LitElement) {
 			});
 	}
 
+	getQuestionKey(record, questionId) {
+		const answer = record && record.Answers && record.Answers.find(a => a.QuestionId === questionId) || false;
+		return answer ? 're-render' : 'render';
```
Comment (BraydonKains) [inline src/components/cpd-record-report.js]: Ahh I forgot about Tom's favourite HTML editor workaround 🤣

**Pull request merged**